packages/storybook/src/nimble/table-column/mapping/table-column-mapping.mdx

```diff
@@ -23,8 +25,8 @@ The <Tag name={tableColumnMappingTag}/> renders specific number, boolean, or str
 ### Best Practices
 
 -   Provide a mapping for every expected record value. Because grouping is performed on the record value, non-mapped record values can result in multiple groups without group labels.
-    -   To improve grouping behavior of values that don't correspond to icons, explicitly map those values to an `undefined` icon.
--   Avoid having multiple values that map to the same label because grouping and sorting is done on the record value rather than the mapped label.
+    -   To improve grouping behavior of a value that shouldn't be rendered in table cells, use a <Tag name={mappingEmptyTag}/> element to provide text for that value's group row.
```

Comment:
I didn't follow the discussion about when to use the empty mapping versus an icon mapping with no icon. Is this always the preferred approach? If so should we update the table-column-mapping story to use an empty mapping instead of the existing "Unknown" icon mapping?

Reply:
We're going to remove support for icon mappings without an icon because we now have the empty mapping element. Therefore, empty mappings will always be the preferred approach. I can go ahead and update the story now, though, since even with both currently supported, the empty mapping would still be preferred.

I'm planning on removing `undefined` icon support from the icon column in a follow-up change since it's a breaking change, and I need to make adjustments to a few dependent apps before that change should go in.